#!/usr/bin/env python
PACKAGE = "blort_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("recovery_conf_threshold", double_t, 0, "Threshold for recovery confidence", 0.05, 0, 0.5)
gen.add("visualize", bool_t, 0, "Visualize object pose", False)
exit(gen.generate(PACKAGE, "gldetector_node", "detector"))
